src/playlist.js

```diff
@@ -100,13 +100,15 @@ const forwardDuration = function(playlist, endSequence) {
   * playlist. The duration of a subinterval of the available segments
   * may be calculated by specifying an end index.
   *
-  * @param {Object} playlist a media playlist object
-  * @param {Number=} endSequence an exclusive upper boundary
-  * for the playlist. Defaults to playlist length.
-  * @return {Number} the duration between the first available segment
+  * @param playlist {object} a media playlist object
```

Comment:
doc comments appear to be backwards (`endSequence {number}` should be `{number} endSequence`) also you can signal optional by putting and equal sign after the type decleration ie `{Number=}`